app/containers/GamePage/index.js

```diff
@@ -99,8 +151,22 @@ export default class GamePage extends Component {
               >
                 Send
               </Button>
+              <Button
+                onClick={this.deleteMessages}
+              >
+                Delete messages
+              </Button>
             </FormGroup>
           </form>
+          <Bang
+            assign={assign}
+            database={database}
+            email={email}
+            params={params}
+            participants={participants}
+            players={players}
```

Comment:
Though if you do like to keep it I would have called it totalPlayers or something. players is too vague